### DISCUSSION TITLE
Details documentation

### DISCUSSION THREAD

**Discussion by smartkanak** [cirruslabs/tart#696] [General]
I am shocked at how good, but at the same time how unknown tart seems to be. Almost nothing on Reddit or YouTube, unfortunately I also find the official documentation very poor (I'm completely new to VM's and have a lot of reading to do). I read through all the discussions on Github today, it's my only source at the moment, but it has already shown me the most important things.

I'm only concerned with the use of VM's, use cases and all sorts of useful functionality, what you would use for what. Have I missed any documentation? Where is the best place to get into this topic. The area of application is not yet defined, I'm just interested in it privately.

But regardless of that: great software, keep up the good work!

**Comment by smartkanak** [comment]
Gonna use --help commands for now

**Comment by fkorotkov** [comment]
Hey @smartkanak, what would you love to see from such a documentation? Introduction to VMs/Containers/etc.? Agreed that Tart is kind of requires previous experience using Docker or some virtualization software before. I'm also wondering what brought you to Tart given that you "completely new to VM's"?

**Reply by smartkanak** [reply]
Some administrative restrictions on my local machine prompted me to explore alternative solutions, allowing me to work and evaluate software freely. While I've already discovered a method to install and run software/CLI tools without sudo, my fascination with VMs persists. It's primarily a personal interest, but the power of this topic makes it a compelling exploration for the future.

**Reply by 1stqualifier** [reply]
@fkorotkov I'm in similar position to @smartkanak I came across tart by complete chance and was instantly hooked and utilise it for personal use. But lack of more detailed documentation is not helping. E.g. it took me a while to identify how to change the default location for stored VM's by using TART_HOME, and even though I now know it I need to run it every time - I don't see this being documented anywhere, it all came from random discussions in random forums. Since you say that "Tart requires previous experience using Docker or some virtualization software before" what would you suggest as a starting point for noobs wanting to start with tart?

**Comment by fkorotkov** [comment]
There is now a great video introduction to Tart. Please check it out:

https://www.youtube.com/watch?v=FLC2WA1OWJY&ab_channel=KorbenTech

**Reply by 1stqualifier** [reply]
Yes, this is the exact video that introduced me to tart - and it was by complete chance as I started watching it because of the creepy AI French to English VoiceOver dub. 
What he covers here is basics available via -h. Still good first step for those new to the idea.
What would be nice is to have documentation on environment variables (e.g. the TART_HOME) and how to use them.
That said, I realise it's a niche tool for people in the know.

**Comment by DrGo** [comment]
https://medium.com/@salah.mahmud/quick-introduction-into-using-tart-for-virtualizing-a-linux-machine-on-a-macos-system-with-apple-88310d6c8cbc
more to come
